src/main/java/wolox/training/models/Book.java

```diff
@@ -158,4 +158,8 @@ public void update (String genre, String author, String image, String title, Str
         this.setYear(year);
         this.setPage(page);
     }
+
+    public void add_user(User user){
```

Comment:
I don't think this method will be used. Once a book is automatically acquired by a user with the declared relationship, the user will be added to the book list.